PayPalMessaging/src/main/java/com/braintreepayments/api/paypalmessaging/PayPalMessagingView.kt

```diff
@@ -0,0 +1,4 @@
+package com.braintreepayments.api.paypalmessaging
+class PayPalMessagingView {
```

Comment:
We can call it `PayPalMessagingClient` - originally in the spike we had gone we `View` because on Swift we are rendering a `UIView` (I don't remember the raw type on Android we'd be using). But I am not tied to this name in any way, so if we feel `Client` is less confusing for merchants I can make that change on both platforms!

Reply:
If it's using a `WebView` view might make sense, but I maybe prefer to not expose underlying implementation details in the naming. I'm good with whatever you all think makes more sense for this one.